index.html

```diff
@@ -38,5 +38,130 @@ <h4>LET'S CONNECT</h4>
+                <div class="lang3 languages">javascript</div>
+            </div>
+            <div class="seeProject">See Project</div>
+        </div>
+        <!-- card 3 -->
+        <div class="card">
+            <img class="image" src="./Images/Snapshoot-page-4.png" alt="tonic">
+            <h2>Tonic</h2>
+            <ul class="triple">
+                <li class="canopy">CANOPY</li>
+                <li class="double">Back End Dev</li>
+                <li class="double" >2015</li>
+            </ul>
+            <p class="paragraph">
+                A daily selection of privately personalised reads; no accounts or no sign-ups required.
+            </p>
+            <div class="langs">
+                <div class="lang1 languages">html</div>
+                <div class="lang2 languages">css</div>
+                <div class="lang3 languages">javascript</div>
+            </div>
+            <div class="seeProject">See Project</div>
+        </div>
+        <!-- card 4 -->
+        <div class="card">
+            <img class="image" src="./Images/Snapshoot-page-5.png" alt="tonic">
+            <h2>Multi-Post Stories</h2>
+            <ul class="triple">
+                <li class="canopy">CANOPY</li>
+                <li class="double">Back End Dev</li>
+                <li class="double">2015</li>
+            </ul>
+            <p class="paragraph">
+                A daily selection of privately personalised reads; no accounts or no sign-ups required.
+            </p>
+            <div class="langs">
+                <div class="lang1 languages">html</div>
+                <div class="lang2 languages">css</div>
+                <div class="lang3 languages">javascript</div>
+            </div>
+            <div class="seeProject">See Project</div>
+        </div>
+    </section>
+    <section class="About Myself">
+        <h1>About<br> Myself</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website
+            Look through some of my work and experience! If you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+        </p>
+        <h4 class="connect">LET'S CONNECT</h4>
+        <ul class="second socials">
+            <li><a href="https://github.com/Christianib003"><img class="social" src="./Images/GutHub.png" alt="github"></a></li>
+            <li><a href="https://www.linkedin.com/in/christian-iradukunda-byiringiro-657598226"><img class="social resize"
+                        src="./Images/png-transparent-computer-icons-social-media-linkedin-slideshare-linkedin-gray-angle-text-rectangle-thumbnail.png"
+                        alt="linkedIn"></a></li>
+            <li><a href="#"><img class="social" src="./Images/M.png" alt="M"></a></li>
+            <li><a href="#"><img class="social" src="./Images/Peace.png" alt="peace"></a></li>
+        </ul>
+        <div class="Resume">Get my resume</div>
```

Comment:
- [ ] Kindly please add a `<button>` tag here instead of a `<div>`.  That way, the element will look and behave like a button. 💪 
- [ ] Good job implementing the font family! 💯  Let's now kindly add some font styles to this element.  Please go to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) and look them up there for this particular element.

Please kindly put special attention to the font size, font-weight, and the line-height.

- [ ] Good job implementing the icons! Now, please kindly let's correct their size and alignment:

Your project:

![image](https://user-images.githubusercontent.com/57421823/148663622-4c9bdca9-e39a-44ee-9c59-d7aef1dc9251.png)

The figma:

![image](https://user-images.githubusercontent.com/57421823/148663632-607db612-3fd1-44eb-96c0-afb2eb05341c.png)

Kindly look at their properties on the template page, so they look more professional and consistent 💯 


That way, this section will look a lot more like the given template 💪